api/service/query_type.go

```diff
@@ -0,0 +1,60 @@
+package service
+
+import (
+	"regexp"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/photon-storage/go-photon/crypto/bls"
+
+	"github.com/photon-storage/photon-explorer/database/orm"
+)
+
+type queryResp struct {
+	Type string `json:"type"`
```

Comment:
The concrete data is to be added later on?

Reply:
this type is to let the front end know which api to call for the concrete data.